app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java

```diff
@@ -52,7 +52,7 @@ protected void onCreate(Bundle savedInstanceState) {
                 R.layout.fragment_step_remove_pages,
                 R.layout.fragment_step_reorder_pages,
                 R.layout.fragment_step_extract_images,
-                R.layout.fragment_step_excel_to_pdf,
+                //R.layout.fragment_step_excel_to_pdf,
```

Comment:
Is the layout "fragment_step_excel_to_pdf" not excel to PDF? 

Also, dont comment, remove it.